### PR TITLE
Test added New manual scripts

### PR DESCRIPTION
**Pull request by kowsisoundhar12**
Added new manual scripts:
    Test_TC_DA_1_3 .yaml
	Test_TC_DIAG_TH_NW_2_2.yaml
	Test_TC_DIAG_TH_NW_2_3.yaml
	Test_TC_DIAG_TH_NW_2_6.yaml
	Test_TC_DIAG_TH_NW_2_7.yaml
	Test_TC_DIAG_TH_NW_2_8.yaml
	Test_TC_DIAG_TH_NW_2_9.yaml
	Test_TC_DM_4_1.yaml
	Test_TC_DM_4_10.yaml
	Test_TC_DM_4_11.yaml
	Test_TC_DM_4_12.yaml
	Test_TC_DM_4_13.yaml
	Test_TC_DM_4_14.yaml
	Test_TC_DM_4_15.yaml
	Test_TC_DM_4_16.yaml
	Test_TC_DM_4_17.yaml
	Test_TC_DM_4_18.yaml
	Test_TC_DM_4_19.yaml
	Test_TC_DM_4_2.yaml
	Test_TC_DM_4_20.yaml
	Test_TC_DM_4_21.yaml
	Test_TC_DM_4_3.yaml
	Test_TC_DM_4_4.yaml
	Test_TC_DM_4_5.yaml
	Test_TC_DM_4_6.yaml
	Test_TC_DM_4_7.yaml
	Test_TC_DM_4_8.yaml
	Test_TC_DM_4_9.yaml
[Manual-scripts-executionlogs.zip](https://github.com/project-chip/connectedhomeip/files/8587856/Manual-scripts-executionlogs.zip)

### CODE REVIEW

**Review by woody-apple — APPROVED** [review]

**Comment by bzbarsky-apple** [comment]
Please do not merge this until after https://github.com/project-chip/connectedhomeip/pull/17905 merges.

**Pull request merged**
